src/explore-education-statistics-common/src/modules/table-tool/components/__tests__/TimePeriodDataTable.test.tsx

```diff
@@ -266,6 +266,7 @@ describe('TimePeriodDataTable', () => {
         subjectName: 'Absence in prus',
         timePeriodRange: [{ label: '2014/15', code: 'AY', year: 2014 }],
         geoJsonAvailable: false,
+        isCroppedTable: false,
```

Comment:
As I mentioned before, we should probably have at least one test here where this flag is true.  Plus test at least once that when this is false, the cropped text *doesn't* appear as well.